spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -361,8 +361,9 @@ func TestSample(t *testing.T) {
 }
 
 func TestBackupSample(t *testing.T) {
-	_ = testutil.EndToEndTest(t)
-
+	if os.Getenv("GOLANG_SAMPLES_E2E_TEST") == "" {
+		t.Skip("GOLANG_SAMPLES_E2E_TEST not set")
+	}
 	id := randomID()
```

Comment:
nit: I think it would be good to call `_ := testutil.SystemTest(t)` here, just as we do in `TestCustomerManagedEncryptionKeys`, to ensure that a project ID has been set.